packages/google_sign_in/google_sign_in_web/lib/src/js_interop/gapi.dart

```diff
@@ -10,15 +10,15 @@
 
 // https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/gapi
 
-// ignore_for_file: public_member_api_docs, unused_element, sort_constructors_first, prefer_generic_function_type_aliases
+// ignore_for_file: public_member_api_docs,
```

Comment:
I'd add below an explanation of why we're ignoring this lint for file (same as in gapiauth2.dart)

Reply:
Done.